lib/sync_decider_context.js

```diff
@@ -67,7 +67,7 @@ SyncDeciderContext.prototype.complete = function(result) {
 };
 
 SyncDeciderContext.prototype.log = function() {
-  console.log.apply(null, arguments);
+  console.log.apply(this, arguments);
```

Comment:
It wasn't printing with `null`, but was with `this`. I couldn't find any docs that told me why though.